File: pset6/mario/less/mario.py
from cs50 import get_int

# we use get_int to make sure the user inputs a int,
# also to convert it from string to int.
height = get_int("Height: ")

# while loop to keep asking the user for a valid positive integer between (1-8)
while (height <= 0 or height > 8):
    height = get_int("Height: ")

for i in range(height):
    print(" " * (height - 1 - i), end = "")
    print("#" * (i + 1))

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{
    //making a variable to store height
    int height;

    //Prompt the user for height in range (1-8) correctly if wrong we keep asking the user again
    do
    {
        height = get_int("Height in range (1-8): ");
    }
    while (height < 1 || height > 8);

    //Navigating the coloumns with variable i
    for (int i = 0; i < height; i++)
    {
        //adding space s times less than the height based on which coloumn we are working on
        for (int s = 1; s < height - i ; s++)
        {
            printf(" ");
        }
        //adding the hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        //print two spaces
        printf("  ");
        //print the hases again
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        //making a new line for the new coloumn
        printf("\n");
    }
}
